src/components/Dashboard/Flavors.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import configureStore from 'redux-mock-store';
+
+import { initialState } from 'reducers/flavors';
+import ConnectedFlavors from './Flavors';
+import { withMemoryRouter } from 'utils';
+
+describe('Dashboard <Flavors />', () => {
+  const defaultOpts = {
+    border: false,
+    header: true,
+    title: false,
+    style: {}
+  };
+  const mockStore = configureStore();
+  const store = mockStore({ flavors: initialState });
+  const RoutedFlavors = withMemoryRouter(ConnectedFlavors);
+
+  it('renders correctly', () => {
+    const component = renderer.create(
```

Comment:
I don't consider this gating, but if you don't need the reference to `component` then you can just do `const tree = renderer.create(<Blah />).toJSON();` and then the expect is a little cleaner.

Reply:
I missed this earlier, probably quite a few more now. I'll correct them (so you don't have to repeat this comment 30 times ;)